src/shogun/mathematics/linalg/internal/implementation/MeanEigen3.h

```diff
@@ -73,11 +75,24 @@ struct int2float<int32_t>
 template <>
 struct int2float<int64_t>
 {
+	/** float64_t type **/
 	using floatType = float64_t;
 };
- 
+#else
```

Comment:
Does cause a build error?
The SWIG #ifdefs are not really used in compilation mode, but only for the swig wrapper (that should not need this type anyways)